server/src/main/java/org/jboss/as/server/deployment/RootDeploymentUnitService.java

```diff
@@ -65,12 +66,33 @@ public RootDeploymentUnitService(final String name, final String managementName,
                                      final ImmutableManagementResourceRegistration registration, final ManagementResourceRegistration mutableRegistration,
                                      final Resource resource, final CapabilityServiceSupport capabilityServiceSupport,
                                      final AbstractVaultReader vaultReader, DeploymentOverlayIndex deploymentOverlays) {
+        this(name, managementName, parent, registration, mutableRegistration, resource, capabilityServiceSupport, vaultReader, deploymentOverlays, false);
```

Comment:
Unless this constructor is used outside of core you don't need to keep the old variant.

If it's used outside of core something is odd. :)